specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimanagement.json

```diff
@@ -0,0 +1,484 @@
+          "items": {
+            "$ref": "#/definitions/PolicySnippetContract"
+          },
+          "description": "Policy snippet value."
+        }
+      },
+      "description": "The response of the list policy snippets operation."
+    },
+    "PolicySnippetContract": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Snippet name.",
+          "readOnly": true
+        },
+        "content": {
+          "type": "string",
+          "description": "Snippet content.",
+          "readOnly": true
+        },
+        "toolTip": {
+          "type": "string",
+          "description": "Snippet toolTip.",
+          "readOnly": true
+        },
+        "scope": {
+          "type": "integer",
+          "description": "Binary OR value of the Snippet scope.",
+          "readOnly": true
+        }
+      },
+      "description": "Policy snippet."
+    },
+    "RegionContract": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Region name.",
+          "readOnly": true
+        },
+        "isMasterRegion": {
+          "description": "whether Region is the master region.",
+          "type": "boolean"
+        },
+        "isDeleted": {
+          "description": "whether Region is deleted.",
+          "type": "boolean"
+        }
+      },
+      "description": "Region profile."
+    },
+    "RegionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegionContract"
+          },
+          "description": "Lists of Regions."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Lists Regions operation response details."
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request."
+    },
+    "SkipQueryParameter": {
+      "name": "$skip",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "format": "int32",
+      "description": "Number of records to skip.",
+      "minimum": 0,
+      "x-ms-parameter-location": "method"
+    },
+    "TopQueryParameter": {
+      "name": "$top",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "format": "int32",
+      "description": "Number of records to return.",
+      "minimum": 1,
+      "x-ms-parameter-location": "method"
+    },
+    "PolicyIdParameter": {
+      "name": "policyId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "policy"
+      ],
```

Comment:
Isnt policy ID provided by the user?

Reply:
no.. it is a static Id 'policy'

---
In reply to: [163335739](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163335739) [](ancestors = 163335739)

Reply:
If that is the case, why not add it already to the paths?

---
In reply to: [163336373](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163336373) [](ancestors = 163336373,163335739)